### ISSUE TITLE
dwc:datasetName

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/happy-hours#36]
## What is your idea for a happy hour topic?
How to use dwc:datasetName. Currently the primary use case is for aggregating specimens associated with a specific project.

## Who could be involved in leading this happy hour?
Active in Slack discussion: @ljwalker, Kesler Randall, Jessica Utrup, Roger Burkhalter, Christina Byrd

## Are there any timing considerations?
_[add text here - e.g. needs to happen before xyz conference, after field season, ASAP, etc.]_

## Next steps
This issue captures a happy hour topic idea. Once you are ready to finalize an idea and put it on the PDWG happy hour schedule, please create a new issue using the [Plan a happy hour](https://github.com/paleo-data/happy-hours/issues/new?assignees=&labels=PDWG&projects=&template=plan-happy-hour.md&title=%5Breplace+this+text+with+the+happy+hour+topic%5D) template. **Then, close this issue.**

**Comment by ekrimmel** [comment]
Closing this issue because the topic is incorporated elsewhere
